<html>
 
<link rel="stylesheet", href="sfoldstyle.css" />
 
<body>
  
<img src="Sfold_logo_banner.png" alt="Sfold Logo" width="1000" height="75">

<p>
Sfold is a software program developed to predict probable 
  <a href="https://en.wikipedia.org/wiki/RNA">RNA </a> 
  <a href="https://en.wikipedia.org/wiki/Nucleic_acid_secondary_structure">secondary structures </a>
  through structure ensemble sampling and centroid predictions [1,2], with a focus on assessment 
  of  RNA target accessibility [3], for the key applications to the rational design of siRNAs [4]  
  for the suppression of gene expressions, and for the identification of targets for regulatory 
  RNAs particularly <a href="https://en.wikipedia.org/wiki/MicroRNA">microRNAs </a> [5,6].  
</p>
 
<h3> <b> Development </b> </h3> 

The core RNA secondary structure prediction algorithm is based on rigorous statistical (stochastic)
sampling of Boltzmann ensemble of RNA secondary structures, enabling statistical characterization
of any local structural features of potential interest to experimental investigators. 
The sampling approach is the focus of a review [7], and its potential was discussed in an 
earlier review [8]. Predictions by Sfold have lead to new biological insights [9-11]. 
The ideas of ensemble sampling and centroids have been adopted by others not only for RNA 
problems, but also for other fundamental problems in computational biology and genomics [12-16]. 
The implementation stochastic sampling has been included in two widely used RNA software, 
RNA structure [17] and the Vienna RNA package [18], which are also based on the Turner RNA 
thermodynamic parameters [19].

<p>
Sfold-based bioinformatics tools for biological applications were developed from successful 
collaboration with  <a href=" 
https://www.wadsworth.org/senior-staff/kathleen-mcdonough">Kathleen McDonough </a> 
on antisense oligos [20], 
<a href="https://www.linkedin.com/in/erasmus-schneider-32865b5">Erasmus Schneider </a> 
on hammerhead ribozymes [21], 
<a href="https://en.wikipedia.org/wiki/Victor_Ambros">Victor Ambros </a> 
on microRNA targeting [6], and <a href="https://medicine.yale.edu/profile/jun-lu/">Jun Lu
</a> on primary microRNA processing [22]. 
</p>
 
<p>
Development of bioinformatics tools and the Sfold software was funded by 
the <a href="https://www.nsf.gov">National Science Foundation (NSF) </a> 
 (grant 0650991, 0200970) and the 
 <a href="https://www.nih.gov">National Institutes of Health (NIH)</a> 
 (grant R01 GM116855, R01 GM068726, R01 GM 099811, R01 GM 138856).
</p>
<h3> <b> Media coverage </b> </h3> 

Sfold and biological application work have been featured on a 
 <a href="https://academic.oup.com/nar/article/31/24/7280/2904423"> <i>Nucleic Acid Res.</i> 
 cover</a>, 
highlighted in <i> Science </i>  NetWatch [23] and <i> Nature </i>  Research Highlights [24], and were 
 reported in a feature article by <i> Genome Technology </i> (now <i> GenomeWeb </i>) [25] and a 
 four-page keynote interview by <i> Research Media </i> [26].

<h3> <b> Distribution  </b> </h3> 

Sfold runs on Linux, and is freely available to the scientific community for non-commercial 
applications, and is available under license for commercial applications. 
Both the source code and the executables are available at GitHub. <br>

<table>
  <tr> 
    <td>Original authors</td>
    <td> <a href="https://www.albany.edu/sph/faculty/ye-ding">Ye Ding</a> and <a href="https://en.wikipedia.org/wiki/Charles_Lawrence_(mathematician)">Charles E. Lawrence</a> </td>
  </tr>
  <tr> 
    <td>Application model developers</td>
    <td>Dang Long and Chaochun Liu</td>
  </tr>
  <tr> 
    <td>Software developers</td>
    <td>Clarence Chan, Adam Wolenc, William A. Rennie and Charles S. Carmack          </td>
  </tr>
  <tr> 
    <td>Initial release date</td>
    <td>April 1, 2003</td>
  </tr>
</table>

<h3> <b> External links </b> </h3> 

•	<a href="https://github.com/Ding-RNA-Lab/Sfold"> Sfold GitHub repository </a> <br>

•	<a href="https://www.healthresearch.org/sfold-software-for-sirna/"> Sfold commercial licensing </a>

<h3> <b> References </b> </h3> 

1.	 Ding, Y; Lawrence, CE (2003). "A statistical sampling algorithm for RNA secondary structure prediction". 
 <i> Nucleic Acids Res. </i>, <b> 31 </b> (24): 7280–301. doi:10.1093/nar/gkg938. PMC 297010. PMID 14654704. <br>
2.	 Ding, Y; Chan, CY; Lawrence, CE (2005). "RNA secondary structure prediction by centroids in a Boltzmann weighted ensemble".
 <i> RNA </i>, <b> 11 </b> (8): 1157–66. doi:10.1261/rna.2500605. PMC 1370799. PMID 16043502. <br>
3.	 Ding, Y; Lawrence, CE (2001). "Statistical prediction of single-stranded regions in RNA secondary structure and 
 application to predicting effective antisense target sites and beyond".
 <i> Nucleic Acids Res. </i>, <b> 29 </b> (5): 1035–46. doi:10.1093/nar/29.5.1034. PMC 29728. PMID 11222752. <br>
4.	Elbashir, SM; Harborth, J; Lendeckel, W; Yalcin, A; Weber, K; Tuschl, T (2001). “Duplexes of 21-nucleotide RNAs 
 mediate RNA interference in cultured mammalian cells”. <i> Nature </i>, <b> 411 </b> (6836):494-8. doi: 10.1038/35078107. <br>
5.	Lee, RC; Feinbaum, RL; Ambros, V (1993). “The C. elegans heterochronic gene lin-4 encodes small RNAs with antisense 
 complementarity to lin-14”. <i> Cell </i>, <b> 75 </b> (5):843-54. doi: 10.1016/0092-8674(93)90529-y. <br>
6.	 Long, D; Lee, R; William, P; Chan, CY; Ambros, V; Ding, Y (2007). "Potent effect of target secondary structure on 
 microRNA function". <i> Nat Struct Mol Biol </i>., <b> 14 </b> (4): 287–94. doi:10.1038/nsmb1226. PMID 17401373. S2CID 650349.<br>
7.	Mathews, D (2006). "Revolutions in RNA secondary structure prediction". <i> J. Mol. Biol.</i>, <b>359 </b> (3): 526–532. 
 doi:10.1016/j.jmb.2006.01.067. PMID 16500677. <br>
8.	 Zucker, M. (2000). "Calculating nucleic acid secondary structure". <i> Curr. Opin. Struct. Biol.</i>, <b> 10</b> (3): 
 303–310. doi:10.1016/s0959-440x(00)00088-9. PMID 10851192. <br>
9.	 Adams, L. (2017). "Pri-miRNA processing: structure is the key". <i>Nature Reviews Genetics </i>, <b> 18 </b> (3): 145. 
 doi:10.1038/nrg.2017.6. PMID 28138147. S2CID 30513706. <br>
10.	 Liu, C., Rennie, W.A., Carmack, C.S., Kanoria, S., Cheng, J., Lu, J. and Ding, Y. (2014) Effects of genetic variations
 on microRNA:target interactions. <i> Nucleic Acids Res.</i>, <b> 42 </b> (15), 9543-52; doi: 10.1093/nar/gku675. 
 PMID: 25081214; PMCID: PMC4150780. <br>
11.	 Shaveta Kanoria, William Rennie, C. Steven Carmack, Jun Lu and Ye Ding (2022). N6- methyladenosine enhances 
 post-transcriptional gene regulation by microRNAs. <i> Bioinformatics Advances </i>, <b> 2 </b> (1), vbab046. <br>
12.	 Huang, F. W.; Qin, Jing; Reidys, Christian M; Stadler, Peter F (2009). "Target prediction and a statistical sampling 
 algorithm for RNA-RNA interaction". <i> Bioinformatics </i>, <b> 26 </b> (2): 175–181. doi:10.1093/bioinformatics/btp635. 
 PMC 2804298. PMID 19910305.<br>
13.	 Harmanchi, Arif Ozgun; Gaurav, Sharma; Mathews, David H (2009). "Stochastic sampling of the RNA structural 
 alignment space". <i>Nucleic Acids Research.</i>, <b>37 </b> (12): 4063–4075. doi:10.1093/nar/gkp276. PMC 2709569. 
 PMID 19429694.<br>
14.	 Hamada, M; Kiryu, H; Mituyama, T; Asai, K (2009). "Prediction of RNA secondary structure using generalized 
 centroid estimators". <i>Bioinformatics </i>, <b> 25 </b> (4): 465–473. doi:10.1093/bioinformatics/btn601. PMID 19095700.<br>
15.	 Carvalho, L. E.; Lawrence, C. E. (2008). "Centroid estimation in discrete high- dimensional spaces with 
 applications in biology". <i> Proc Natl Acad Sci.</i>, <b> 105 </b> (9): 3209–14. Bibcode:2008PNAS..105.3209C. 
 doi:10.1073/pnas.0712329105. PMC 2265131. PMID 18305160. <br>
16.	 Newberg, L. A.; Thompson, W. A.; Colan, S; Smith, T. M.; McCue, L. A.; Lawrence, C. E. (2007). "Centroid 
 estimation in discrete high- dimensional spaces with applications in biology". <i> Bioinformatics </i>, 
 <b> 23 </b> (14): 1718–27. doi:10.1093/bioinformatics/btm241. PMC 2268014. PMID 17488758.<br>
17. Bellaousov, S; Reuter, JS; Seetin, MG; Mathews, DH (2013). ”RNAstructure: web servers for RNA secondary structure
 prediction and analysis”. <i> Nucleic Acids Res. </i>, <b> 41 </b> (Web Server issue): W471–W474. <br>
18. Gruber, AR; Lorenz, R; Bernhart, SH; Neuböck, R; Hofacker, IL (2008). “The Vienna RNA Websuite”. 
 <i> Nucleic Acids Res. </i>, <b> 36 </b> (Web Server issue): W70–W74.<br>
19. Mathews, DH; Sabina, J; Turner DH (1999).“ Expanded sequence dependence of thermodynamic parameters improves 
 prediction of RNA secondary structure”. <i> J. Mol Biol. </i>, <b> 288 </b> (5):911-40. doi: 10.1006/jmbi.1999.2700.<br>
20. Shao Y, Wu Y, Chan CY, McDonough K, Ding Y (2006). Rational design and rapid screening of antisense oligonucleotides 
 for prokaryotic gene modulation. <i> Nucleic Acids Res. </i>, <b> 34 </b> (19):5660-9.<br>
21. Shao, Y., Wu, S., Chan, C.Y., Klapper, J.R., Schneider, E. and Ding, Y. (2007) A structural analysis of in vitro
 catalytic activities of hammerhead ribozymes (2007). <i> BMC Bioinformatics </i>, <b> 8 </b>: 469.<br>
22. Roden, C.A., Gaillard, J., Kanoria, S., Rennie, W., Barish, S., Cheng, J., Pan, W., Liu, J., Cotsapas, C., 
 Ding, Y., Lu, J. (2017) Novel determinants of mammalian primary microRNA processing revealed by systematic 
 evaluation of hairpin-containing transcripts and human genetic variation. <i> Genome Res. </i>, <b> 27 </b> (1) 
 doi: 10.1101/gr.208900.116. PMID: 28087842; PMCID: PMC5340965.<br>
23. "TOOLS: Nucleic Acid Origami". <i> Science </i>, <b> 300 </b> (5621): 873. 2003. doi:10.1126/science.300.5621.873d. 
 S2CID 220109027. <br>
24. "Cell Biology: Predictions on target". <i> Nature </i>, <b> 446 </b>, 587, 2007, Research Highlights 
 <https://www.nature.com/articles/446586a#Cell-Biology> <br>
25. "Surefire siRNAs". <i>  Genome Technology </i> (now <i> Genomeweb </i>), July 1, 2003, Meredith W Salisbury
  <https://www.genomeweb.com/archive/surefire-sirnas#.Y0gvFC-B28c> <br>
26. "The Sfold Project: Computational tools for regulatory RNA studies". International Innovation. 
   Bristol, UK: <i> Research Media Ltd </i>. February 2010. pp. 22-25. <br>


</body>
</html>  
